test/base/src/androidTest/kotlin/io/realm/test/shared/RealmTests.kt

```diff
@@ -402,6 +406,93 @@ class RealmTests {
             }
     }
 
+    @Test
+    @Suppress("LongMethod")
+    fun deleteRealm() {
+        val fileSystem = FileSystem.SYSTEM
+        val testDir = PlatformUtils.createTempDir("test_dir")
+        val testDirPath = testDir.toPath()
+        assertTrue(fileSystem.exists(testDirPath))
+
+        val configuration = RealmConfiguration.Builder(schema = setOf(Parent::class, Child::class))
+            .path("$testDir/default.realm")
```

Comment:
This shouldn't compile anymore? 🤔 
```suggestion
            .directory(testDir)
```